### ISSUE TITLE
pudb barfs if $HOME isn't set

### ISSUE THREAD

**Issue by bukzor** [inducer/pudb#145]
I sometimes need to debug in a reduced-environment ... environment where this variable isn't set.

pudb starts up fine, but subsequently crashes with "Permission denied: /.config".

I'd expect pudb to just skip config persistence in this case, possibly with a warning.

**Comment by inducer** [comment]
Can you cook up a patch?

**Comment by bukzor** [comment]
Almost certainly.

Where do I add the regression test?

**Comment by inducer** [comment]
See #40. :-1:

**Comment by lechat** [comment]
I think that setting default to ``/`` if home is not set on line 4 in settings.py will result in this message.

Possibly falling back to $TEMP first and then to current dir would be the easiest thing to implement.

**Comment by lechat** [comment]
Added pull request for this: #151

**Comment by bukzor** [comment]
Agree. Fallback should be None, and the patch consists primarily of handling the None gracefully.

**Comment by lechat** [comment]
I think that this issue could be closed now.

**Comment by inducer** [comment]
@bukzor : Agree?
